heron/uploaders/src/java/com/twitter/heron/uploader/scp/ScpContext.java

```diff
@@ -0,0 +1,38 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.twitter.heron.uploader.scp;
+
+import com.twitter.heron.spi.common.Config;
+import com.twitter.heron.spi.common.Context;
+
+public class ScpContext extends Context {
+  public static final String HERON_UPLOADER_SCP_COMMAND = "heron.uploader.scp.command";
```

Comment:
The value of this constant is not same as the one used in config file. It is missing the substring ".options". Is that ok?